Description: Upstream changes introduced in version 0.5-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 python-ethtool (0.5-1) unstable; urgency=low
 .
   * Packaging for Debian main (Closes: #549323).
 .
 The person named in the Author field signed this changelog entry.
Author: Miroslav Suchý <miroslav@suchy.cz>
Bug-Debian: http://bugs.debian.org/549323

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ python-ethtool-0.5/man/pifconfig.8.asciidoc
@@ -0,0 +1,42 @@
+pifconfig(8)
+============
+
+NAME
+----
+pifconfig - display information about a network interface
+
+
+SYNOPSIS
+--------
+pifconfig [INTERFACE]
+
+
+DESCRIPTION
+-----------
+
+This script mimic ifconfig behavior, but is written purely in python
+and python module ethtool.
+
+
+OPTIONS
+-------
+
+INTERFACE::
+Display information about only this interface. If no interface is given
+then all interfaces are displayed.
+
+-h, --help::
+Show help message and exit.
+
+
+SEE ALSO
+--------
+ifconfig(8)
+
+
+
+AUTHORS
+-------
+Arnaldo Carvalho de Melo <acme@redhat.com>
+
+Man page written by Miroslav Suchý <msuchy@redhat.com>
--- /dev/null
+++ python-ethtool-0.5/man/pethtool.8.asciidoc
@@ -0,0 +1,81 @@
+pethtool(8)
+===========
+
+NAME
+----
+pethtool - Display or change ethernet card settings
+
+
+SYNOPSIS
+--------
+pethtool [OPTIONS] [<INTERFACE>]
+
+
+DESCRIPTION
+-----------
+
+This script mimic ethtool behavior, but is written purely in python
+and python module ethtool.
+
+
+OPTIONS
+-------
+
+INTERFACE::
+Is the name of the ethernet device on which pethtool should operate.
+
+-h, --help::
+Show help message and exit.
+
+-c|--show-coalesce::
+Show coalesce options
+
+-C|--coalesce::
+Set coalesce options
+
+                adaptive-rx on|off
+                adaptive-tx on|off
+                rx-usecs N
+                rx-frames N
+                rx-usecs-irq N
+                rx-frames-irq N
+                tx-usecs N
+                tx-frames N
+                tx-usecs-irq N
+                tx-frames-irq N
+                stats-block-usecs N
+                pkt-rate-low N
+                rx-usecs-low N
+                rx-frames-low N
+                tx-usecs-low N
+                tx-frames-low N
+                pkt-rate-high N
+                rx-usecs-high N
+                rx-frames-high N
+                tx-usecs-high N
+                tx-frames-high N
+                sample-interval N
+
+
+-i|--driver::
+Show driver information
+
+-k|--show-offload::
+Get protocol offload information
+
+-K|--offload::
+    Set protocol offload;;
+        [ tso on|off ]
+
+
+
+SEE ALSO
+--------
+ethtool(8)
+
+
+AUTHORS
+-------
+Arnaldo Carvalho de Melo <acme@redhat.com>
+
+Man page written by Miroslav Suchý <msuchy@redhat.com>
